gulpfile.js

```diff
@@ -35,26 +36,36 @@ gulp.task('sass', function() {
 });
 
 gulp.task('dist:css', function() {
-  return gulp.src('_sass/hamburgers/hamburgers.scss')
-    .pipe(sass())
-    .pipe(autoprefixer(autoprefixerOptions))
-    .pipe(gulp.dest('dist'))
-    .pipe(csscomb('.csscomb.dist.json'))
-    .pipe(cssnano())
-    .pipe(rename('hamburgers.min.css'))
-    .pipe(gulp.dest('dist'));
+  var types = [    '3dx',    '3dx-r',    '3dy',    '3dy-r',    '3dxy',    '3dxy-r',    'arrow',    'arrow-r',    'arrowalt',    'arrowalt-r',    'arrowturn',    'arrowturn-r',    'boring',    'collapse',    'collapse-r',    'elastic',    'elastic-r',    'emphatic',    'emphatic-r',    'minus',    'slider',    'slider-r',    'spin',    'spin-r',    'spring',    'spring-r',    'stand',    'stand-r',    'squeeze',    'vortex',    'vortex-r'  ];
+
+  var streams = types.map(function(type) {
+    return gulp.src('_sass/hamburgers/hamburgers.scss')
+      .pipe(sass({
+        data: '@import "types/' + type + '";'
+      }))
+      .pipe(autoprefixer(autoprefixerOptions))
+      .pipe(gulp.dest('dist'))
+      .pipe(csscomb('.csscomb.dist.json'))
+      .pipe(cssnano())
+      .pipe(rename('hamburgers-' + type + '.min.css'))
+      .pipe(gulp.dest('dist'));
+  });
+
+  return merge(streams);
 });
 
+
 gulp.task('watch', function() {
   var browserSyncConfig = require('./bs-config.js');
 
   browserSync.init(browserSyncConfig);
 
-  gulp.watch('_sass/**/*.scss', ['sass']);
+  gulp.watch('_sass/**/*.scss', gulp.series('dist:css'));
 });
```

Comment:
Update to [gulp.series] to make it compatible with gulp v4. Consider push a change to the origin repo